integration-tests/meteor/docker-compose.yml

```diff
@@ -46,17 +53,24 @@ services:
       - testapp1:8080
       - '--'
       - /bin/oplogtoredis
+
   mongo:
-    image: mongo:4.4
+    image: mongo:5.0.19
     command: "mongod --replSet myapp --port 27017"
     logging:
       driver: none
+    healthcheck:
+      test: test $$(echo "rs.initiate({_id:'myapp',members:[{_id:0,host:\"mongo:27017\"}]}).ok || rs.status().ok" | mongo --port 27017 --quiet) -eq 1
+      interval: 10s
+      start_period: 10s
```

Comment:
`start_period` is sometimes 30s, sometimes 10s - are those necessary differences?

Reply:
I just played around with it while having an issue. I don't know what the ideal `interval` and `start_period` should be.